libs/search-widget/src/components/viewer/renderers/VideoRenderer.svelte

```diff
@@ -0,0 +1,64 @@
+<script lang="ts">
+  import type { Observable } from 'rxjs';
+  import { currentThumbnail, getFieldUrl, getFileUrls, getPlayableVideo, isLinkField, playFrom } from "../../../core";
```

Comment:
weird, there are double quotes on some of these `import`s…

Reply:
it's my IDE which is putting them automatically when updated the imports… I manually put back single quotes, but I might have missed some. I'll try to clean them all up before marking the PR ready for review